dags/process_zip_dag.py

```diff
@@ -201,6 +216,9 @@ def update_tiff_references_to_jpeg_in_article(**context) -> bytes:
         element.attrib[XLINK_HREF] = re.sub(r'\.\w+$', '.jpg', element.attrib[XLINK_HREF])
         element.attrib['mime-subtype'] = 'jpeg'
 
+    for element in article_xml.xpath('//*[contains(@xlink:href, ".tif")]', namespaces=XLINK):
+        element.attrib[XLINK_HREF] = re.sub(r'\.\w+$', '.jpg', element.attrib[XLINK_HREF])
```

Comment:
consider extracting the two `for` loops into two functions to clarify what they are doing